docs/gateway.md

```diff
@@ -2,132 +2,140 @@
 
-Suppose you maintain a container `your-org/a-container-name`. Your users pull your container via `docker pull your-org/a-container-name` (or an alternative container runtime command). Scarf provides a thin redirect layer in front of your registry so that you can offer a registry-independent route to your container, on your own domain.
+Suppose you maintain a Docker container image `acme/rocket-skates` on Docker Hub. Your users would normally pull your container image from Docker hub directly (`docker pull acme/rocket-skates`.) Using Scarf, they would pull from Scarf Gateway using your private domain (`docker pull docker.acme.com/acme/rocket-skates`.) Important analytics are recorded and shared with you while you are using Scarf and you may change your hosting providers at any time without changing your container image name or documentation.
 
-With Scarf, your container can stay on its current registry, but be served through your own domain, e.g.:
+Your users will always be using your domain to pull your Docker container images. Your Docker container images can stay on their current hosting provider, but be served through your own domain, e.g.:
 
 ```bash
 # Make your existing container available through your own domain
-$ docker pull registry.custom.com/your-org/a-container-name
+$ docker pull docker.acme.com/acme/rocket-skates
 
-# Scarf provides a URL too if you'd prefer
-$ docker pull org.docker.scarf.sh/your-org/a-container-name
+# Scarf provides a domain too if you'd prefer to use ours
+$ docker pull acme.docker.scarf.sh/acme/rocket-skates
 ```
 
-Data insights about your container's downloads can be found in your Scarf dashboard. From there, you can also manage your gateway configuration, access controls, and more.
+Data insights about your Docker container image's downloads can be found in your Scarf Dashboard. From there, you can also manage Scarf Gateway configuration, access controls, and more.
 
 ### Configuring
 
 **Packages**
 
-Every container served through the Scarf Gateway needs a corresponding _package_ entry on [scarf.sh](scarf.sh). Configuration, analytics, and permissions are all done at the level of a package, or single repository. `hello-world`, `myorg/image` are all valid package entries. Because packages can seamlessly change their registry, hostnames (e.g. `gcr.io`) are not part of the package identifier, i.e. `myorg/image` and not `gcr.io/myorg/image`. 
+Every Docker container image served through Scarf Gateway needs a corresponding _package_ entry on [scarf.sh](scarf.sh). Configuration, analytics, and permissions are all done at the level of a package, or single repository [ED: this may be seen as a negative]. `rocket-skates`, `acme/rocket-skates` are all valid package entries. Because packages can seamlessly change their hosting provider, hostnames (e.g. `gcr.io`) are not part of the package identifier on Scarf, i.e. `acme/rocket-skates` and not `gcr.io/acme/rocket-skates`.
```

Comment:
Did you mean to leave this ED?

Reply:
yes. until it's solved. easily removed

Reply:
Maybe let's reword then to say that we are investigating workarounds. No need to publicly criticize ourselves for something out of our control :)

Reply:
It was a minor comment but "Configuration, analytics, and permissions are all done at the level of a package, or single repository [ED: this may be seen as a negative]." is the context. I meant that having to configure all this for every package may seem tedious.  I'll remove the ED sentence.